changelogs/unreleased/tnt_tx_push.md

```diff
@@ -0,0 +1,7 @@
+## feature/box
```

Comment:
Should probably be `feature/core`.

I also wouldn't go too deep into details in the release notes. For example, I wouldn't mention the configuration parameter - it should go to the documentation.

Reply:
Done.